examples/simple/posix/UB-PTHREAD5-good.c

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) 2015 Runtime Verification, Inc. (RV-Match team). All Rights Reserved.
+
+#include <pthread.h>
+#include <stdlib.h>
+#include <string.h>
+#include <stdio.h>
+
+pthread_mutex_t lock;
+
+int main(void)
+{
+    
```

Comment:
any reason for this whiteline?  you typically didn;t have whitelines at the beginning of functions.  also, not clear to me when you put one whiteline and when you put more.  no big deal, but in general it is a good idea to follow consistent formatting and rules, because that shows seriousness and professionalism.